pkg/flow/flow.go

```diff
@@ -112,6 +112,9 @@ type Options struct {
 	// OnExportsChange is nil, export configuration blocks are not allowed in the
 	// loaded config file.
 	OnExportsChange func(exports map[string]any)
+
+	// IsModule is set to true if the Flow Controller is a module.
+	IsModule bool
```

Comment:
Do we want to make this another option, or do we want to assume that ControllerID being non empty means the controller is for a module?